reverse.py

```diff
@@ -0,0 +1,40 @@
+from string import punctuation, digits
+
+def reversed_func(text):
+    """
+    The function reverses the letters in all words of input text.
+    The order of words preserves.
+    All non-letter symbols/numbers stays on the same places.
+    """
+    PRESERVED_CHARS = punctuation + digits
```

Comment:
You also see it more as a constant value for such occasions. I would suggest moving it outside of the function so we do not re-initialize this variable each time